xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1399,3 +1386,31 @@ DEBUG_INFO_RENDER DX::DeviceResources::GetDebugInfo() const
 
   return info;
 }
+
+std::vector<DXGI_COLOR_SPACE_TYPE> DX::DeviceResources::GetSwapChainColorSpaces() const
+{
+  if (!m_swapChain)
+    return std::vector<DXGI_COLOR_SPACE_TYPE>();
```

Comment:
```suggestion
    return {};
```
Is also valid and seems more simple